config.template.yml

```diff
@@ -91,6 +91,13 @@ authentication_backend:
   # than one instance and therefore is recommended for
   # production.
   ldap:
+    # The LDAP implementation, this affects elements like the attribute utilised for resetting a password.
+    # Acceptable options are as follows:
+    # - 'activedirectory' - For Microsoft Active Directory.
+    # - 'rfc' - For most open source LDAP implementations, for example rfc2307.
+    # This currently defaults to 'rfc' to maintain existing behaviour.
+    # implementation: rfc
```

Comment:
```suggestion
    # This currently defaults to 'custom' to maintain existing behaviour.
    # implementation: custom
```

Reply:
More of a what do you think? Since it affects the password reset function maybe not.

Reply:
Happy with whatever really here.

My thought process around having rfc as the default would be implementing the second part of this change.

Which as we discussed would be to include default ldap settings for attributes and filters when selecting a specific implementation, which also can be overridden if the options are present.